client/src/components/TextForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import { Header, Segment, Container, Button, Form } from "semantic-ui-react";
+import languageOptions from "./LanguageOptions";
+
+class AddDocumentForm extends Component {
```

Comment:
Make sure the filename is the same with the component name, you can rename the file "AddDocumentForm.js" here.